### ISSUE TITLE
No clippy output from cargo clippy on macOS

### ISSUE THREAD

**Issue by djc** [rust-lang/rust-clippy#4985]
With clippy 0.0.212 (c8e3cfbd 2019-10-28) on macOS, I'm not getting any output from clippy, even after touching `src/lib.rs`. This happens even while clippy running on GitHub Actions' `ubuntu-latest` shows plenty of warnings in the same project (in this case, the [askama_shared](https://github.com/djc/askama/tree/master/askama_shared) crate). The only output on my MacBook Pro is

```
    Finished dev [unoptimized + debuginfo] target(s) in 0.05s
```

This is with the stable toolchain installed from rustup.

I'm sorry if this is extremely obvious or if there is a prior bug about this that I missed (I did search the issue tracker for a bit).

**Comment by flip1995** [comment]
That is weird. Does `cargo clean` help?

**Comment by djc** [comment]
That does seem to have helped in this case, yes.

**Comment by djc** [comment]
I had the same problem in another crate, so this doesn't seem to be specific to this crate.

**Comment by flip1995** [comment]
Ok, then this is a known problem. This will be fixed, once Clippy is integrated into cargo #3837. In the mean time you can use `cargo clippy-preview`.
